plugins/spark/spark-environment.xml.in

```diff
@@ -0,0 +1,1250 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
is this an exact copy of another environment.xml + sparkthor entries? if so, we shouldn't duplicate the environment.

Reply:
Yes, it's just the default xml + the sparkthor stuff.  Should I remove this file then?